root/admin/attributes/Index.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * @flow
+ * eslint-disable flowtype/no-mutable-array
+ * Copyright (C) 2019 Anirudh Jain
+ * Copyright (C) 2019 MetaBrainz Foundation
```

Comment:
Sorry for not having been clear, it should be:
 * Copyright (C) 2014 MetaBrainz Foundation
 * Copyright (C) 2019 Anirudh Jain